test/Bridges/Constraint/indicator.jl

```diff
@@ -53,3 +53,92 @@ include("../utilities.jl")
         @test t.coefficient ≈ 1.0
     end
 end
+
+@testset "Indicator by SOS1" begin
+    # linear problem with indicator constraints
```

Comment:
Can you also split the file in two so that we have the same files in `src` and `test` folders ?

Reply:
Done in last commit

Reply:
of course not, it's much more fun when it all stays local :upside_down_face:

Reply:
(just did)